test/functional/unified-spec-runner/schema.ts

```diff
@@ -27,25 +27,37 @@ export const TopologyType = Object.freeze({
   single: 'single',
   replicaset: 'replicaset',
   sharded: 'sharded',
-  shardedReplicaset: 'sharded-replicaset'
-} as const);
+  shardedReplicaset: 'sharded-replicaset',
+  loadBalanced: 'load-balanced'
+}) as const;
```

Comment:
```suggestion
} as const);
```
the `as const` cast has to be on the object literal

Reply:
Updated.